src/device-state/target-state.ts

```diff
@@ -70,10 +73,16 @@ export const update = async (
 ): Promise<void> => {
 	await config.initialized;
 	return Bluebird.using(lockGetTarget(), async () => {
-		const { uuid, apiEndpoint, apiTimeout } = await config.getMany([
+		const {
+			uuid,
+			apiEndpoint,
+			apiTimeout,
+			currentApiKey,
```

Comment:
```suggestion
			deviceApiKey,
```
`currentApiKey` was a hack that will be removed at some point.

Reply:
good to know